lio-http-server/src/LIO/HTTP/Server/Responses.hs

```diff
@@ -0,0 +1,148 @@
+{-# LANGUAGE Trustworthy #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Taken from Simple 
```

Comment:
I think we can leave the simple comment without the `-- |` so it doesn't get generated in the haddock html output